packages/opentelemetry-shim-opentracing/src/shim.ts

```diff
@@ -98,6 +111,18 @@ export class SpanContextShim extends opentracing.SpanContext {
   toSpanId(): string {
     return this._spanContext.spanId;
   }
+
+  getBaggageItem(key: string): string | undefined {
+    const entry: api.EntryValue = this._correlationContext[key];
+    return entry === undefined ? undefined : entry.value;
+  }
+
+  setBaggageItem(key: string, value: string) {
+    this._correlationContext = Object.assign(
+      { [key]: { value } },
```

Comment:
I haven't found anything related on the opentracing specification, but seeing different implementations it seems like the correct behaviour is to override the value of the baggage.

Reply:
That was my understanding as well.

Reply:
by saying it is gonna be overwritten I meant this
```
setBaggageItem('foo', 'bar1');
getBaggageItem('foo'); // bar1
setBaggageItem('foo', 'bar2');
getBaggageItem('foo'); // bar1 instead of 'bar2'
```
because you are settings new item and then overwrite with the previous value

Reply:
ok I see you fixed that, just double checking that is the desired behaviour after @dyladan mentioned this